From 65ae0808f4f8053b4d6f885a18535c6be8124eaf Mon Sep 17 00:00:00 2001
From: William Lai <b04597@freescale.com>
Date: Tue, 16 Jun 2009 14:32:19 +0800
Subject: [PATCH 100/435] ENGR00113379 ASRC: Change the output clock source in
 the ASRC test application

1. Change back the output clock source to ASRCK1 clock, as no SSI1 TX clock
	is available unless the SGTL5000 is open.

Signed-off-by: William Lai <b04597@freescale.com>
---
 test/mxc_asrc_test/autorun-asrc.sh | 2 +-
 test/mxc_asrc_test/mxc_asrc_test.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/mxc_asrc_test/autorun-asrc.sh b/test/mxc_asrc_test/autorun-asrc.sh
index 17ccbc6..553c987 100755
--- a/test/mxc_asrc_test/autorun-asrc.sh
+++ b/test/mxc_asrc_test/autorun-asrc.sh
@@ -51,7 +51,7 @@ run_testcase " aplay -N -M /dev/s441.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " rm /dev/s882.wav /dev/s441.wav /dev/raw.txt"
+run_testcase " rm /dev/s882.wav /dev/s441.wav"
 fi
 print_status
 exit $STATUS
diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index 6f62855..ef19801 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -97,8 +97,8 @@ int configure_asrc_channel(int fd_asrc, struct audio_info_s *info)
 	config.word_width = 32;
 	config.inclk = INCLK_NONE;
 	//config.inclk = INCLK_SSI1_TX;
-	config.outclk = OUTCLK_SSI1_TX;
-	//config.outclk = OUTCLK_ASRCK1_CLK;
+	//config.outclk = OUTCLK_SSI1_TX;
+	config.outclk = OUTCLK_ASRCK1_CLK;
 	pair_index = req.index;
 	if (err = ioctl(fd_asrc, ASRC_CONFIG_PAIR, &config) < 0)
 		return err;
-- 
1.8.0

